medusa/providers/torrent/json/ncore.py

```diff
@@ -0,0 +1,176 @@
+# coding=utf-8
+
+"""Provider code for nCore."""
+
+from __future__ import unicode_literals
+
+import logging
+
+from medusa import tv
+from medusa.helper.common import (
+    convert_size,
+    try_int,
+)
+from medusa.logger.adapters.style import BraceAdapter
+from medusa.providers.torrent.torrent_provider import TorrentProvider
+
+from requests.utils import dict_from_cookiejar
+
+log = BraceAdapter(logging.getLogger(__name__))
+log.logger.addHandler(logging.NullHandler())
+
+
+class NcoreProvider(TorrentProvider):
+    """Norbits Torrent provider."""
+
+    def __init__(self):
+        """.Initialize the class."""
+        super(NcoreProvider, self).__init__('nCore')
+
+        # Credentials
+        self.username = None
+        self.password = None
+
+        # URLs
+        self.url = 'https://ncore.cc'
+        self.urls = {
+            'login': 'https://ncore.cc/login.php',
+            'search': 'https://ncore.cc/torrents.php',
+        }
+
+        # Proper Strings
+        self.proper_strings = ['PROPER', 'REPACK', 'REAL', 'RERIP']
+
+        # Miscellaneous Options
+
+        # Cache
+        self.cache = tv.Cache(self, min_time=20)
+
+    def search(self, search_strings, age=0, ep_obj=None, **kwargs):
```

Comment:
You can remove age and ep_obj. I don't think you need to put *args. The **kwargs should pick them up

Reply:
It just shadows how it is implemented in generic_provider. If we need to change it, we should change it for every other provider as well.